drivers/clock_control_nrfxlib/clock_control_nrfxlib.c

```diff
@@ -168,22 +193,65 @@ void nrf_power_clock_isr(void)
 		power_event_cb(NRF_POWER_EVENT_USBREMOVED);
 	}
 #endif
+
+#ifdef CONFIG_CLOCK_CONTROL_BT_LL_NRFXLIB
 	/* FIXME/NOTE: This handler is called _after_ the USB registers are
```

Comment:
As a note. The BLE controller no longer touches the USB register. Can be fixed separately to create a smaller diff